src/_scss/pages/search/searchSidebar.scss

```diff
@@ -8,37 +8,79 @@
     box-shadow: $container-shadow;
     border-radius: rem(5);
     margin-top: rem(32);
+
     .sidebar-header {
         display: none;
-        @include media($tablet-screen) {
-            @include display(flex);
-            @include align-items(center);
-            @include flex-flow(row);
+        @include media($small-screen) {
+            display: inline-table;
+            @include align-items(left);
             @include justify-content(flex-start);
             position: relative;
             border-bottom: rem(1) solid $color-gray-lighter;
             height: rem(70);
+            width: 100%;
             margin: 0;
             padding: 0 0 0 rem(15);
-            & .sidebar-title {
-                @include h6;
-                @include flex(1 1 auto);
-                color: $color-gray;
-                font-size: rem(19);
-                line-height: rem(21);
-                font-weight: $font-semibold;
-                margin: 0;
-            }
-            & .filter-icon {
```

Comment:
something about changes to this class has caused an irregularity in the other instance of this class
![20230314AccountFilterIcon](https://user-images.githubusercontent.com/49243108/225080553-54b30a27-f654-43a1-a852-ad72f25a749a.png)